src/compat/libc/stdlib/wctomb.c

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file
+ * @brief wctomb function.
+ *
+ * @see stdlib.h
+ *
+ * @date 07.12.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+int wctomb(char *out,  const wchar_t *in) {
+	if (in == NULL || out == NULL) {
+		return 0;
+	}
+	if (*in == L'\0') {
+		return 0;
```

Comment:
Single null-char should be written in this case.

Reply:
fixed